casper/src/test/scala/coop/rchain/casper/helper/HashSetCasperTestNode.scala

```diff
@@ -141,6 +142,16 @@ class HashSetCasperTestNode[F[_]](
     )
     result.map(_.right.get)
   }
+
+  def printVisualizeDagUrl(): F[Unit] =
+    for {
+      dot <- visualizeDag()
+      // Java's URLEncode encodes ' ' as '+' instead of '%20', see https://stackoverflow.com/a/5330239/52142
+      urlEncoded = URLEncoder.encode(dot, "UTF-8").replaceAll("\\+", "%20")
+      _ <- Sync[F].delay {
+            println(s"DAG @ $name: https://dreampuf.github.io/GraphvizOnline/#" + urlEncoded)
```

Comment:
Maybe add a comment saying that the whole dag is embedded in the URL: it wasn't intuitive at first